enigma-crypto/src/symmetric.rs

```diff
@@ -15,10 +15,14 @@ use crate::localstd::vec::Vec;
 use crate::localstd::vec;
 use crate::rand;
 
-const IV_SIZE: usize = 96/8;
 static AES_MODE: &aead::Algorithm = &aead::AES_256_GCM;
-type IV = [u8; IV_SIZE];
 
+/// The symmertic key byte size
+pub const SYMMETRIC_KEY_SIZE: usize = 32;
```

Comment:
What is the purpose of this constant in this file if it is not used in the code?

Reply:
It's used in `runtime_t`. I'm just trying to avoid re-defining constants that can be important. Let me know if there's a better place to put shared constants.

Reply:
Added directly in `runtime_t` for now. Please let me know what is the right place for this constant.